src/Courses/CourseDetailsTabs/OverrideGradeModal.tsx

```diff
@@ -227,25 +239,41 @@ export const OverrideGradeModal: React.FC<OverrideGradeModalProps> = ({
                         switch(overrideGradePhase) {
                         case OverrideGradePhase.PROMPT:
                             return (
-                                <Button variant="primary" type="submit">
+                                <Button
+                                    variant="primary"
+                                    type="submit"
+                                    // The input gets focus instead of the button
+                                    // ref={(ref: HTMLButtonElement | null) => ref?.focus()}
+                                >
                                         Submit
                                 </Button>
                             );
                         case OverrideGradePhase.CONFIRM:
                             return (
-                                <Button variant="primary" onClick={overrideGradeConfirm}>
+                                <Button variant="primary"
+                                    onClick={overrideGradeConfirm}
+                                    ref={(ref: HTMLButtonElement | null) => ref?.focus()}
```

Comment:
Should these just have the `autofocus` attribute instead? I think it's a native element attribute.

Reply:
I tried didn't work

We talked in slack and you sent across some great links

https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/742
https://github.com/facebook/react/issues/11851#issuecomment-351780994
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/742#issuecomment-691318099

Reply:
Skipping for now